packages/react-core/src/components/SimpleList/SimpleListItem.tsx

```diff
@@ -15,7 +17,7 @@ export interface SimpleListItemProps {
   /** Additional props added to the SimpleList <a> or <button> */
   componentProps?: any;
   /** Indicates if the link is current/highlighted */
-  isCurrent?: boolean;
```

Comment:
Unfortunately, changing the API - changing prop names - on public facing components constitutes a breaking change. So I agree this name change is a good idea, but we cannot be making breaking changes to our components unless it's a breaking change release.

Reply:
It's a beta component. I don't know if it's allowed, but @tlabaj told me so. I can of course revert the name back.

Reply:
@nicolethoen That problem is caused by `component="a" href="#"` in the `SimpleListItem`. I don't know what was the reasoning behind this, it causes refresh of the page, so state is lost...

Reply:
Yes, this is still a beta component so we have flexibility to make some improvements to the API.